README.md

```diff
@@ -34,14 +34,236 @@ VSCode allows for development within a docker container. This project has an alr
 
 -[Developing inside a Container](https://code.visualstudio.com/docs/remote/containers) -[Docker](https://www.docker.com/)
 
+
+### Prerequisites
+
+Make sure the following dependencies are installed on your system:
+- Poetry (1.5.1)
+- Visual Studio Code
+- Docker
+- MangoDB Compass
+- 1Password
+
 ## Getting Started
 
-### To Begin Development
+Follow these steps to set up the development environment after installing the prerequisites.
+1. Setting up GitHub SSH Auth and Signing Keys and configure SSH authentication and add your SSh public key to your GitHub account.
```

Comment:
This is still missing what we did to set up the 1pass agent for ssh keys.

Reply:
I take this back, it looks like you put the steps out of order. The 1pass agent should be the very first thing you do, you cannot create ssh keys without it.